lib/bolt/transport/local/connection.rb

```diff
@@ -68,7 +68,10 @@ def execute(command)
             end
           end
 
-          Open3.popen3(*command)
+          # It honestly seems like we should just always do this.
+          Bundler.with_unbundled_env do
```

Comment:
I think we would have to start shipping the bundler gem with bolt for this.

Reply:
Isn't bundler a default gem?

Reply:
Gah didn't even think of that...I think I can wrap this in a check that Bundler is available and not do this if it's not, since it'll be moot in that case.

Reply:
Boo, looks like it's only default starting with ruby 2.6. :( https://stdgems.org/compare/